ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -85,6 +85,7 @@ object Main extends StrictLogging {
         config.applicationId,
         config.address,
         config.httpPort,
+        Config.DefaultWsConfig,
```

Comment:
you don't allow overriding the default WS settings. I have not found the logic that reads `WebsocketConfig` from command line arguments? Is that intentional, kept for a separate PR?

Reply:
yeah, I did not add logic to pass config yet. wanted to get the PR up for feedback first.